spec/licensee/content_helper_spec.rb

```diff
@@ -39,7 +39,7 @@ def initialize(content = nil)
     let(:content) { 'license' * 1000 }
 
     it 'returns the max delta' do
-      expect(subject.max_delta).to eql(140)
+      expect(subject.max_delta).to eql(280)
```

Comment:
Should we add a test to ensure a short license has the expected max delta?

Reply:
Added.